mteb/models/gritlm.py

```diff
@@ -14,6 +14,7 @@ def gritlm_loader(**kwargs):
         raise ImportError(
             "GritLM is not installed. Please install it with `pip install gritlm`."
         )
+    kwargs.pop("device") # GritLM does automatic device placement
```

Comment:
Could we add a check?
```suggestion
    if "device" in kwargs:
      kwargs.pop("device") # GritLM does automatic device placement
```

Reply:
I think kwargs.pop does not fail if the text is not in kwargs but just return None in that case, so fine without checking, no?

Reply:
Oh right! Maybe like this?
```suggestion
    kwargs.pop("device", None) # GritLM does automatic device placement
```

I was reading [here](https://docs.python.org/3/library/stdtypes.html#dict.pop) that 
> If default is not given and key is not in the dictionary, a KeyError is raised.

Reply:
Oh you're right; let's go with that then!